microlith/Dockerfile

```diff
@@ -4,6 +4,13 @@ FROM prom/prometheus:v2.28.1 as prometheus-official
 FROM prom/alertmanager:v0.22.2 as alertmanager-official
 FROM jaegertracing/all-in-one:1.25.0 as jaegaer-official
 
+# Generate documentation and include locally as a static site
+FROM antora/antora:3.0.0-alpha.1 as antora-generator
+COPY docs/ /antora/docs/
+RUN mkdir -p /antora/cmos
+ENV HOME=/antora
+CMD ["--to-dir", "/antora/cmos/", "antora-playbook.yaml"]
```

Comment:
Shouldn't there be a `RUN` somewhere here? I'm not seeing how Antora is actually executed